BREEZE.rst

```diff
@@ -1046,15 +1046,21 @@ This is the current syntax for  `./breeze <./breeze>`_:
 
   Detailed usage for command: static-check
 
-  breeze [FLAGS] static-check -- <EXTRA_ARGS>
+  breeze [FLAGS] static-check <STATIC_CHECK> -- <EXTRA_ARGS>
 
         Run selected static checks for currently changed files. You should specify static check that
         you would like to run or 'all' to run all checks. One of:
 
-                 all all-but-pylint bat-tests check-apache-license check-executables-have-shebangs
-                 check-hooks-apply check-merge-conflict check-xml debug-statements doctoc
-                 detect-private-key end-of-file-fixer flake8 forbid-tabs insert-license
-                 lint-dockerfile mixed-line-ending mypy pylint pylint-test setup-order shellcheck
+                 all all-but-pylint airflow-config-yaml base-operator bat-tests build
+                 build-providers-dependencies check-apache-license check-executables-have-shebangs
+                 check-hooks-apply check-integrations check-merge-conflict check-xml
+                 consistent-pylint daysago-import-check debug-statements detect-private-key doctoc
+                 end-of-file-fixer fix-encoding-pragma flake8 forbid-tabs
+                 incorrect-use-of-LoggingMixin insert-license isort lint-dockerfile mixed-line-ending
+                 mypy provide-create-sessions pydevd pylint pylint-tests python-no-log-warn
+                 rst-backticks setup-order shellcheck stylelint trailing-whitespace
+                 update-breeze-file update-extras update-local-yml-file update-setup-cfg-file
+                 yamllint
```

Comment:
Does it work with autocomplete?

Reply:
Yes. that's the whole point of having ti as pre-commit does not have auto-complete.